cmd/yggctl/actions.go

```diff
@@ -42,6 +56,81 @@ func generateControlMessageAction(c *cli.Context) error {
 	return nil
 }
 
+func messageJournalAction(ctx *cli.Context) error {
+	// Set the worker message truncate length from the user-provided arguments
+	truncateLength := ctx.Int("truncate-message")
+	if truncateLength <= 0 {
+		return cli.Exit(
+			fmt.Errorf(
```

Comment:
go-staticcheck reports a linting suggestion here:

`error strings should not end with punctuation or newlines (ST1005)go-staticcheck`

Reply:
Thanks! Looks like there's another linting tool I didn't have enabled :+1: